app/components/Footer/index.js

```diff
@@ -19,9 +19,10 @@ function Footer() {
         <FormattedMessage
           {...messages.authorMessage}
           values={{
-            author: <A href="https://twitter.com/mxstbr">Max Stoiber</A>,
+            author: <A onClick={() => {}}>Max Stoiber</A>,
```

Comment:
There are still some limitations, see here where the same issue isn't flagged because its nested inside of a FormattedMessage value, but it can add additional value to our automated tools